<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EnvelopeForm</class>
 <widget class="QWidget" name="EnvelopeForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>160</width>
    <height>87</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_26">
     <property name="text">
      <string>Initial volume</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSpinBox" name="mInitVolumeSpin">
     <property name="maximum">
      <number>15</number>
     </property>
     <property name="value">
      <number>15</number>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QCheckBox" name="mIncreasingCheckbox">
     <property name="text">
      <string>Increasing</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_27">
     <property name="text">
      <string>Period</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSpinBox" name="mPeriodSpin">
     <property name="maximum">
      <number>7</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
